bootloaders/uart/uart_binary.c

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "pico/stdlib.h"
+#include "hardware/uart.h"
+#include "hardware/structs/pads_qspi.h"
+#include "hardware/structs/io_qspi.h"
+
+#ifndef LED_DELAY_MS
+#define LED_DELAY_MS 500
```

Comment:
From what I understand, I think the "parent" RP2350 will reset the "child" RP2350 if it doesn't receive the string "Hello" once per second; so I guess that means that `LED_DELAY_MS` **can't** be larger than 500?